native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -107,7 +123,12 @@ static int fdGetter(const std::string &url, off_t *start, off_t *length) {
         fd = AAsset_openFileDescriptor(asset, start, length);
         AAsset_close(asset);
     }
-
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+    FileUtilsOpenHarmony* fileUtils = dynamic_cast<FileUtilsOpenHarmony*>(FileUtils::getInstance());
```

Comment:
If using dynmaic_cast, do we need to check whether the return value is nullptr?
If not, I think static_cast is enough.

Reply:
Yes, using cc_assert requires checking for downcasting.